imagefudge/image_fudge.py

```diff
@@ -34,33 +34,41 @@ def draw_relative_arcs(self, origins, endpoints, arclen):
             provide support for true negative arclen
         """
 
+        try:
+            if arclen > 360 and arclen < -360:
+                raise ValueError('arclen must be between -360 and 360')
+            arclen_is_range = False
+        except TypeError as te:
+            try:
+                if arclen[0] > 360 and arclen < -360 or\
+                   arclen[1] > 360 and arclen < -360:
+                    raise ValueError(('arclen must be between'
+                                      ' -360 and 360'))
+                arclen_is_range = True
```

Comment:
Would that be more efficient than a flag though? The flag should only be set once per function call. Passing in an integer value _should_ run faster but I haven't timed it